dump/plugins/portchannel.py

```diff
@@ -0,0 +1,113 @@
+from dump.match_infra import MatchRequest
+from dump.helper import create_template_dict
+from .executor import Executor
+
+
+class Portchannel(Executor):
+    """
+    Debug Dump Plugin for PortChannel/LAG Module
+    """
+    ARG_NAME = "portchannel_name"
+
+    def __init__(self, match_engine=None):
+        super().__init__(match_engine)
+        self.ret_temp = {}
+        self.ns = ''
+        self.lag_members = set()
+
+    def get_all_args(self, ns=""):
+        req = MatchRequest(db="CONFIG_DB", table="PORTCHANNEL", key_pattern="*", ns=ns)
+        ret = self.match_engine.fetch(req)
+        all_lags = ret["keys"]
+        return [key.split("|")[-1] for key in all_lags]
+
+    def execute(self, params_dict):
+        self.ret_temp = create_template_dict(dbs=["CONFIG_DB", "APPL_DB", "ASIC_DB", "STATE_DB"])
+        self.lag_name = params_dict[Portchannel.ARG_NAME]
+        self.ns = params_dict["namespace"]
+        # CONFIG_DB
+        lag_found = self.init_lag_config_info()
+        if lag_found:
+            self.init_lag_member_config_info()
```

Comment:
how it lag member details used in dump state of port channel? 
Can you provide sample output of the cli 'dump state portchannel ' which is using port channel member data?

Reply:
I'm collecting the lag_members into the `self.lag_members` object. and then using the information in the `init_lag_member_type_obj_asic_info` method to find the SAI_OBJECT_TYPE_LAG object.